sdk/samples/iot/hub/README.md

```diff
@@ -157,6 +157,10 @@ Start using the IoT Hub Client in your solutions. More SDK details can be found
 
 For extensive documentation on Azure IoT Hub, see the [API reference documentation][iot_hub_mqtt].
 
+## Contributing
+
+This project welcomes contributions and suggestions. Find [more contributing][SDK_README_CONTRIBUTING] details here.
```

Comment:
I don't think `SDK_README_CONTRIBUTING` is a link below?

Reply:
whoops lol